https/README.md

```diff
@@ -0,0 +1,47 @@
+# https Package for prometheus
+
+The `https` directory contains files and a template config for the implementation of tls.
+When running a server with tls use the flag --web.tls-config=" /path to config/.yml "
```

Comment:
the spaces here will cause issues. The example below is sufficient, just mention the flag name here.